src/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './Checkbox.scss';
+import { CHECKBOX_VIEWBOX } from '../../constants';
+
+const { block, elem } = bem({
+    name: 'Checkbox',
+    classnames: styles,
+    propsToMods: ['disabled']
+});
+
+const Checkbox = props => {
+    const { id, children, viewbox, disabled, ...rest } = props;
+
+    return (
+        <label {...block(props)} htmlFor={id} disabled={disabled}>
+            <input
+                type="checkbox"
+                id={id}
+                {...rest}
+                {...elem('input', props)}
+                disabled={disabled}
+            />
+            <span {...elem('box', props)}>
+                <svg {...elem('svg', props)} width="12px" height="10px" viewBox={viewbox}>
+                    <polyline points="1.5 6 3.5 9 8 3" />
+                </svg>
+            </span>
+            <span {...elem('text', props)}>{children}</span>
+        </label>
+    );
+};
+
+Checkbox.propTypes = {
```

Comment:
This should probably have an `onChange` callback function, otherwise you cannot capture the checkstate

Reply:
Now that I'm thinking about it once more, considering this structure any additional attributes from `...rest` would be applied to `<label />` while `onChange` should be applied to `<input />` AFAICT. Can you please double check the implementation so that we're sure that a callback can be fired (also including the checkstate as argument) when the checkstate has changed?

Reply:
I added a story to demonstrate it